cmd/mountlib/mount.go

```diff
@@ -199,7 +199,7 @@ normal drives to be fast and reliable, while cloud storage is far from
 that.  See also [Limitations](#limitations) section below for more
 info
 
-Add "--fuse-flag --VolumePrefix=\server\share" to your "mount"
+Add ` + "`--fuse-flag --VolumePrefix=\\server\\share`" + ` to your ` + "`mount`" + `
```

Comment:
I don't think these \ need to be doubled up? I can see them displaying correctly here: https://tip.rclone.org/commands/rclone_mount/ (this is the docs website made from master).

Reply:
Without it, Go was complaining about unknown escape sequences.

I don't know Go, but my understanding is that \`...\` is a literal string, so you can put anything in it without escaping.  But "..." is a normal string, so "\s" is seen as the escape sequence for "s", which doesn't exist.

Reply:
resolved in the rebased tree